### PR TITLE
[IMP] general: companies rewrite

### PR DESCRIPTION
**Pull request by tiku-odoo**
Tasks:

https://www.odoo.com/web#id=3443150&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

https://www.odoo.com/web#id=3459544&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### CODE REVIEW

**Comment by tiku-odoo** [comment]
@jero-odoo 

This doc is ready for your review when you have a moment. 

Thanks, 
Tim :+1:

File: content/applications/general/companies.rst
Comment (jero-odoo) [inline content/applications/general/companies.rst:117]:
```suggestion
Company`. Multiple companies can be set for :guilabel:`Allowed Companies`, and only one can be set
as the :guilabel:`Default Company`.
```

File: content/applications/general/companies.rst
Comment (jero-odoo) [inline content/applications/general/companies.rst:113]: Early line break, less than 100 characters

File: content/applications/general/companies.rst
Comment (jero-odoo) [inline content/applications/general/companies.rst:165]: Early line break, less than 100 characters

File: content/applications/general/companies.rst
Comment (tiku-odoo) [reply content/applications/general/companies.rst:200]: No this is a general note

**Comment by tiku-odoo** [comment]
@jero-odoo 

I've made all your requested changes. Can you approve?

Thanks,
Tim

**Review by jero-odoo — APPROVED** [review]
Looks good @tiku-odoo ! Approving so this can go to the next round :+1:

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:194]:
```suggestion
   US`, automatically creates a vendor bill, and generates a drafted purchase/sales order using the
```

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:195]:
```suggestion
   selected company warehouse, when a sales/purchase order is confirmed for the selected company. If,
```

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:197]:
```suggestion
   instead of a drafted purchase/sales order, it should be validated, enable
   :guilabel:`Automatic Validation`.
```

**Review by ksc-odoo — CHANGES_REQUESTED** [review]
@tiku-odoo -- just finished my Final Review. Please address my comments, and tag me again for another look. Thanks.

File: content/applications/general/companies.rst
Comment (tiku-odoo) [reply content/applications/general/companies.rst:105]: I've added clarification. 

Thanks for spotting this.

File: content/applications/general/companies.rst
Comment (tiku-odoo) [reply content/applications/general/companies.rst:151]: I've added in clarification and some more details.

**Comment by tiku-odoo** [comment]
@ksc-odoo 

This doc is ready for your second review. I've made all the changes that you suggested. 

Thanks! 

Tim

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:22]:
```suggestion
   If they are on a *Yearly* or *Multi-year* contract, this creates an upsell order, with a 30-day
```

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:27]:
```suggestion
   functionality is still detected while they are on a *Standard* plan, they are automatically switched,
```

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:41]:
```suggestion
   :guilabel:`Companies` page is not in list view, click the :guilabel:`≣ (four bars)` icon, located in
```

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:71]:
```suggestion
   Companies also have a :guilabel:`Parent Company` set on the company form in :ref:`developer
```

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:84]:
```suggestion
   environment the database is in belongs to: *My Company (San Francisco)*.
```

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:103]:
```suggestion
When an environment is selected from the top menu, along with an additional company, records are
```

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:124]:
```suggestion
   Branches also have a :guilabel:`Parent Company` set on the branch form in :ref:`developer mode
```

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:127]:
```suggestion
   menu, and go to :menuselection:`Settings app --> Accounting --> Fiscal Localization`.
```

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:136]:
```suggestion
Employee access
```

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:156]:
```suggestion
If an administrator has multiple companies activated on the database, and is editing a record, the
```

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:166]:
```suggestion
- The current company selected in the company selector, in the upper-right hand of the screen (the
```

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:175]:
```suggestion
- The company set is the company linked to the document (the same as if a record is being edited)
```

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:180]:
```suggestion
To set document formats according to each company, activate and select the respective company,
```

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:182]:
```suggestion
edit the information, as needed.
```

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:188]:
```suggestion
:guilabel:`Company Details` can be edited on the document layout. By default, this field is populated
```

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:189]:
```suggestion
from the company information listed, when navigating here: :menuselection:`Settings app --> Companies section -->
```

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:209]:
```suggestion
With the respective company activated and selected, choose one of the following options:
```

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:211]:
```suggestion
- :guilabel:`Do not synchronize`: do not synchronize any inter-company transactions.
```

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:225]: i'm assuming the / is make sure the asterisk appears as a footnote, and not just an italicized sentence, correct?

Comment (ksc-odoo) [inline content/applications/general/companies.rst:225]:
```suggestion
  \*The given option needs to be selected, so :guilabel:`Automatic Validation` appears in the
```

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:239]:
```suggestion
   selected company warehouse, when a sales/purchase order is confirmed for the selected company. If,
```

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:240]:
```suggestion
   instead of a drafted purchase/sales order, it should be validated, enable
```

File: content/applications/general/companies.rst
Comment (ksc-odoo) [inline content/applications/general/companies.rst:245]:
```suggestion
   confirmed, if the :guilabel:`Automatic Validation` feature was enabled).
```

**Review by ksc-odoo — APPROVED** [review]
great work implementing my suggested changes, @tiku-odoo -- found a few more during this run through -- but i'm approving now, because I don't anticipate you having any issues making the necessary changes. when this is ready, feel free to tag it for Tech Review. Thanks 👍

File: content/applications/general/companies.rst
Comment (tiku-odoo) [reply content/applications/general/companies.rst:225]: Yes that is correct on the asterisk

**Comment by tiku-odoo** [comment]
@samueljlieber 

This script is ready for your tech review when you have a moment. 

Thanks,
Tim

File: content/applications/general/companies.rst
Comment (samueljlieber) [inline content/applications/general/companies.rst:44]: On the UI the button is labeled "⚙️ Actions" and not "Action ⚙️", no need for icon descriptor when an icon is accompanied by a label :)
```suggestion
   After selecting the appropriate company, click the :guilabel:`⚙️ Actions` icon, and select
```

File: content/applications/general/companies.rst
Comment (samueljlieber) [inline content/applications/general/companies.rst:67]: No need to format _social media_ IMO
```suggestion
   Activate the :ref:`developer mode <developer-mode>` to set social media accounts and
```

File: content/applications/general/companies.rst
Comment (samueljlieber) [inline content/applications/general/companies.rst:72]: early break
```suggestion
   Companies also have a :guilabel:`Parent Company` set on the company form in :ref:`developer mode
   <developer-mode>`.
```

File: content/applications/general/companies.rst
Comment (samueljlieber) [inline content/applications/general/companies.rst:120]:
```suggestion
   Activate the :ref:`developer mode <developer-mode>` to set social media accounts and
```

File: content/applications/general/companies.rst
Comment (samueljlieber) [inline content/applications/general/companies.rst:182]: Suggestion to stay consistent with the navigation path throughout the doc :)
```suggestion
and, under the :menuselection:`Settings app --> Companies section`, click on :guilabel:`Configure
Document Layout` and edit the information as needed.
```

File: content/applications/general/companies.rst
Comment (samueljlieber) [inline content/applications/general/companies.rst:192]: Sentence case header please :)
```suggestion
Inter-company transactions
```

File: content/applications/general/companies.rst
Comment (samueljlieber) [inline content/applications/general/companies.rst:195]: The bold **First** formatting seems a little strange to me, but I understand it is calling attention that debug mode **must** be activated in order to see the _Inter-Company Transactions_ tab. 

Maybe there is a better way to structure this notice just before navigating to the _Inter-Company Transactions_ tab since debug mode doesnt necessarily need to be enabled to configure the Chart of Accounts, Taxes, etc...?
```suggestion
First, activate the :ref:`developer mode <developer-mode>`. Then, make sure each one of the
```

File: content/applications/general/companies.rst
Comment (samueljlieber) [inline content/applications/general/companies.rst:207]: Suggestion to move the navigation instructions before the tab configuration, as this is a more logical flow when reading through.
```suggestion
Next, navigate to :menuselection:`Settings app --> Companies section --> Manage Companies`. Then,
select the desired company from the list. On the company form, select the :guilabel:`Inter-Company
Transactions` tab, on the individual company's detail form.
```

File: content/applications/general/companies.rst
Comment (samueljlieber) [inline content/applications/general/companies.rst:209]: Add the field name for specificity 
```suggestion
With the respective company activated and selected, choose one of the following :guilabel:`Rule`
options:
```

File: content/applications/general/companies.rst
Comment (samueljlieber) [inline content/applications/general/companies.rst:216]: Add a space and escape symbol 
```suggestion
  sales order, it should be validated, enable :guilabel:`Automatic Validation`. \*
```

File: content/applications/general/companies.rst
Comment (samueljlieber) [inline content/applications/general/companies.rst:219]:
```suggestion
  drafted purchase order, it should be validated, enable :guilabel:`Automatic Validation`. \*
```

File: content/applications/general/companies.rst
Comment (samueljlieber) [inline content/applications/general/companies.rst:223]:
```suggestion
  Validation`. \*
```

File: content/applications/general/companies.rst
Comment (samueljlieber) [inline content/applications/general/companies.rst:226]: If this notice applies to all the * points, lets break it out from the last bullet point (also add a space).
```suggestion
\* The given option needs to be selected, so :guilabel:`Automatic Validation` appears in the
configuration.
```

File: content/applications/general/companies.rst
Comment (samueljlieber) [inline content/applications/general/companies.rst:251]: Update to relative path
```suggestion
   - :doc:`Multi-company Guidelines <../../developer/howtos/company>`
```

File: content/applications/general/companies/access_rights_multi_companies.png
Comment (samueljlieber) [inline content/applications/general/companies/access_rights_multi_companies.png:1]: Image file should have hyphens rather than underscores

File: content/applications/general/companies/document_layout.png
Comment (samueljlieber) [inline content/applications/general/companies/document_layout.png:1]: Image file should have hyphens rather than underscores

File: content/applications/general/companies/inter_company_transactions.png
Comment (samueljlieber) [inline content/applications/general/companies/inter_company_transactions.png:1]: Image file should have hyphens rather than underscores

File: content/applications/general/companies/multi_companies_menu_dashboard.png
Comment (samueljlieber) [inline content/applications/general/companies/multi_companies_menu_dashboard.png:1]: Image file should have hyphens rather than underscores

File: content/applications/general/companies/product_form_company.png
Comment (samueljlieber) [inline content/applications/general/companies/product_form_company.png:1]: Image file should have hyphens rather than underscores

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @tiku-odoo, nice job on this improvement to the Companies doc! I think this will be a great improvement for linking from localization docs. I have a small handful of technical changes and suggestions for you, please see below.

The following images are unused in this doc and can be removed:
- `favicon.png`
- `invoice_inter_company.png`
- `purchase_order_inter_company.png`

Please tag me for another quick look once these are addressed, thank you!

File: content/applications/general/companies.rst
Comment (tiku-odoo) [reply content/applications/general/companies.rst:192]: This was a change from KC.

**Comment by tiku-odoo** [comment]
@samueljlieber 

This doc is ready for another look when you have a chance. Thanks! 

Tim

**Review by samueljlieber — COMMENTED** [review]
Looks good to me @tiku-odoo, nice job 👍
...
@robodoo r+

**Pull request closed without merging**
